xbmc/dbwrappers/sqlitedataset.cpp

```diff
@@ -129,56 +238,17 @@ int SqliteDatabase::status(void) {
   return DB_CONNECTION_OK;
 }
 
-int SqliteDatabase::setErr(int err_code, const char * qry){
-  switch (err_code) {
-  case SQLITE_OK: error ="Successful result";
-    break;
-  case SQLITE_ERROR: error = "SQL error or missing database";
-    break;
-  case SQLITE_INTERNAL: error = "An internal logic error in SQLite";
-    break;
-  case SQLITE_PERM: error ="Access permission denied";
-    break;
-  case SQLITE_ABORT: error = "Callback routine requested an abort";
-    break;
-  case SQLITE_BUSY: error = "The database file is locked";
-    break;
-  case SQLITE_LOCKED: error = "A table in the database is locked";
-    break;
-  case SQLITE_NOMEM: error = "A malloc() failed";
-    break;
-  case SQLITE_READONLY: error = "Attempt to write a readonly database";
-    break;
-  case SQLITE_INTERRUPT: error = "Operation terminated by sqlite_interrupt()";
-    break;
-  case  SQLITE_IOERR: error = "Some kind of disk I/O error occurred";
-    break;
-  case  SQLITE_CORRUPT: error = "The database disk image is malformed";
-    break;
-  case SQLITE_NOTFOUND: error = "(Internal Only) Table or record not found";
-    break;
-  case SQLITE_FULL: error = "Insertion failed because database is full";
-    break;
-  case SQLITE_CANTOPEN: error = "Unable to open the database file";
-    break;
-  case SQLITE_PROTOCOL: error = "Database lock protocol error";
-    break;
-  case SQLITE_EMPTY:  error = "(Internal Only) Database table is empty";
-    break;
-  case SQLITE_SCHEMA: error = "The database schema changed";
-    break;
-  case SQLITE_TOOBIG: error = "Too much data for one row of a table";
-    break;
-  case SQLITE_CONSTRAINT: error = "Abort due to constraint violation";
-    break;
-  case SQLITE_MISMATCH:  error = "Data type mismatch";
-    break;
-  default : error = "Undefined SQLite error";
-  }
-  error = "[" + db + "] " + error;
-  error += "\nQuery: ";
-  error += qry;
-  error += "\n";
+int SqliteDatabase::setErr(int err_code, const char * qry) {
+  std::stringstream ss;
```

Comment:
Small code style issue - `{` on new line here and elsewhere

Reply:
The whole file is non-conforming so I don't think it makes much sense to change that on this line

Reply:
Fair enough